plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveTestUtils.java

```diff
@@ -180,6 +180,8 @@ public static Set<HivePageSourceFactory> getDefaultHivePageSourceFactories(HdfsE
     public static Set<HivePageSourceFactory> getDefaultHivePageSourceFactories(TrinoFileSystemFactory fileSystemFactory, HiveConfig hiveConfig)
     {
         FileFormatDataSourceStats stats = new FileFormatDataSourceStats();
+        // set IonNativeTrino as true in hiveConfig for testing
+        hiveConfig.setIonNativeTrinoEnabled(true);
```

Comment:
I do not know, but it strikes me that this is the only config getting set _here_.
So either:
1. No other HiveConfig is getting set for any of the tests.
2. Other formats are setting it somewhere else for their tests.

But I really doubt 1 is true. What do you see?

Reply:
I tried to find what other formats are doing but I think no other format has something like a feature flag which requires an explicit opt-in. They have all their default values as expected. Rather for us we need to explicitly enable this flag, which is why I have it set here.